applications/GeoMechanicsApplication/tests/test_prescribed_derivatives.py

```diff
@@ -0,0 +1,101 @@
+# from KratosMultiphysics import * as Kratos
+
+import sys
+import os
+
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.GeoMechanicsApplication as KratosGeo
+import KratosMultiphysics.GeoMechanicsApplication.geomechanics_analysis as analysis
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+import test_helper
+
+
+class KratosGeoMechanicsPrescribedDerivatives(KratosUnittest.TestCase):
+    """
+    This class contains tests which check if prescribed derivatives are correctly applied to the model and not overwritten
+    """
+
+    def setUp(self):
+        # Code here will be placed BEFORE every test in this TestCase.
+        pass
+
+    def tearDown(self):
+        # Code here will be placed AFTER every test in this TestCase.
+        pass
+
+    def test_prescribed_acceleration(self):
```

Comment:
If there are any other things to test besides the prescribed variables themselves, please let me know!